cmd/cli/cli.go

```diff
@@ -1,9 +1,13 @@
-//
-// Copyright (c) 2020-present Snowplow Analytics Ltd. All rights reserved.
-//
-// This program is licensed to you under the Snowplow Community License Version 1.0,
-// and you may not use this file except in compliance with the Snowplow Community License Version 1.0.
-// You may obtain a copy of the Snowplow Community License Version 1.0 at https://docs.snowplow.io/community-license-1.0
+/**
```

Comment:
In the old version, `//`s was used but I guess `/** */` does work as well.

Reply:
Yeah makes no difference and easier copy paste to just use exactly what's in the other repos :D